### DISCUSSION TITLE
Is it possible to "add Products"?

### DISCUSSION THREAD

**Discussion by CookieMonster760** [jef/streetmerchant#3116] [Help]
I was wondering if i can add Products like Mainbords or Casings. If this is possible can someone please help me to get this right? And Yes i searched for id and found this link "https://jef.codes/streetmerchant/help/general/#adding-a-store" but it is not valid anymore. Can someone help?

**Comment by rasurbrand** [comment]
https://www.jef.buzz/streetmerchant/help/general/
Here you will find everything

**Comment by Mr-Tech-13** [comment]
Don't forget to add 

config.ts:
'series': envOrArray(process.env.DISCORD_NOTIFY_GROUP_SERIES),
and
'series': envOrNumber(process.env.MAX_PRICE_SERIES_SERIES),

dotenv:
DISCORD_NOTIFY_GROUP_SERIES=
MAX_PRICE_SERIES_(SERIES)=

Replace series with what you are adding. To add links see below:

import {Store} from './store';

export const (store): Store = {
  currency: '(can be anything in https://github.com/jef/streetmerchant/blob/main/src/store/model/store.ts#L313)',
  labels: {
    inStock: {
      container:
        '(copied selector)',
      text: ['(add to cart text)'],
    },
    outOfStock: {
      container:
        '(out of stock selector. Can be anything that says out of stock)',
      text: ['(out of stock text)'],
    },
  },
  links: [
    {
      brand: 'test:brand',
      model: 'test:model',
      series: 'test:series',
      url: 'https://www.power.no/data-og-tilbehoer/p-1128968/',
    },
//   VV  Copy and paste as many times as needed but don't leave this one blank! VV
    {
      brand: '',
      model: '',
      series: '',
      url: '',
    },
